### DISCUSSION TITLE
Any clever way to extract the main target photo (plane) in this image

### DISCUSSION THREAD

**Discussion by ardieMejia** [ImageMagick/ImageMagick#6280] [Help] [answered]
### ImageMagick version

7.1.0-43

### Operating system, version and so on

Ubuntu 18.04.1

### Description

Is there any clever way to extract only the plane photo in this image (original image). I know how to remove the white padding around the brown paper (second image). So any suggestions from either starting point is welcome. I want to know any before attempting a rather complicated method of looping through lines.  
![14_55171106-720](https://user-images.githubusercontent.com/47635526/233953555-693a971d-7887-4246-b6ac-1d7dc99e8416.jpg)
![output](https://user-images.githubusercontent.com/47635526/233953600-64ee7d4f-c1c5-4921-aa63-7f57ee994c7d.png)

**Comment by snibgo** [comment, marked as answer]
What is visually different about the plane photo, compared to the rest of the scan? It contains large dark areas. So we can use any process that finds the boundary of large dark areas.

Naming your image `pilot.jpg`, Windows BAT syntax:
```
magick ^
  pilot.jpg ^
  -colorspace Gray ^
  -auto-level ^
  -blur 0x10 ^
  -threshold 65%% ^
  x2.png
```
`x2.png` looks like this:
![x2](https://user-images.githubusercontent.com/33812368/234036952-adf5d2e7-34c8-431b-bc1b-a28c726c82b4.png)

`connected-components` can be used to ignore small components. This input has only one large component. If there is more than one, a script can find the bounding box of all the bounding boxes. The blur means the result won't give exactly the correct crop, so you should add a margin to the numbers, and crop to those numbers, and you will probably need a final trim.

**Reply by ardieMejia** [reply]
Thanks! Wasnt expecting a reply this early! Always nice to know, there are coders out there still using IM. Ill start from your suggestion, its already lots of help. So im marking it as an answer.
